frontend/src/components/eam/eam.tsx

```diff
@@ -58,6 +75,35 @@ const EAM: FC = () => {
     );
   };
 
+  const handleLoadWorkers = (from: number): void => {
+    dispatch(
+      eamActions.loadWorkers({
+        tenantId: tenantId as string,
+        from: from,
+        count: 5,
+      }),
+    );
+  };
+
+  const handleBackPage = (): void => {
```

Comment:
since we will use pagination for a lot of pages, we need to create a custom hook for it